generated/nidaqmx/system/storage/persisted_task.py

```diff
@@ -135,9 +135,10 @@ def load(self):
         Returns:
             nidaqmx.task.Task: Indicates the loaded Task object.
         """
-        task_handle = self._interpreter.load_task(self._name)
+        task_handle = None
```

Comment:
This is not an `__init__` function, so we don't have to worry about `__del__` here. Delete this line.